review/npm.md

```diff
@@ -309,9 +309,13 @@ unpinned dependencies and update the lockfiles:
       calculator](https://semver.npmjs.com/) to help you define the ranges.
 
    1. The lockfile `package-lock.json` ***should*** be ignored for tests running in CI
-      (e.g. via `npm install --no-package-lock`). The reasoning is that developers should
+      (e.g. via `npm install --no-package-lock`). The reasoning is that CI tests should
```

Comment:
to be fair tho, this is also true for developers, despite the (relatively small) risk to their local machines.

Reply:
I don't entirely disagree. I think developers will do in practice regardless of what we say (e.g. when troubleshooting). I think the intention is to raise awareness and encourage users to run these non-locked tests in CI/less-privileged env when possible. 

Let me know if you think we should change it or if this is acceptable as-is.